pinaxcon/registrasion/management/commands/djangogirls.py

```diff
@@ -0,0 +1,77 @@
+import csv
+
+from django.core.management.base import BaseCommand
+from django.contrib.auth.models import User
+from pinaxcon.registrasion.models import AttendeeProfile
+from registrasion.controllers.cart import CartController
+from registrasion.controllers.invoice import InvoiceController
+from registrasion.models import Voucher
+from registrasion.models import conditions
+from registrasion.models import Attendee
+from registrasion.models import Product
+from registrasion.models import Invoice
+
+
+class Command(BaseCommand):
+    help = "Registers a list of people with a django girls ticket. If someone already has a ticket, won't change that."
+
+    def add_arguments(self, parser):
+        parser.add_argument('--csv', help='CSV input, must include: email, name, food_needs, access_needs columns.')
+
+    def handle(self, *args, **options):
+
+        djangoticket = Product.objects.filter(name="DjangoGirls")[0]
+        voucher = Voucher.objects.filter(recipient="DjangoGirls")[0]
+
+        details = []
+        with open(options['csv'], 'r') as csvfile:
+            reader = csv.DictReader(csvfile, fieldnames=None)
+            for row in reader:
+                details.append(row)
+
+        for detail in details:
+            email = detail['email']
+
+            user, created = User.objects.get_or_create(username=email, email=email)
+
+            print("%s user %s" % (email, created_or_found(created)))
+
+            attendee, created = Attendee.objects.get_or_create(user=user)
+
+            print("%s attendee %s" % (email, created_or_found(created)))
+
+            profile, created = AttendeeProfile.objects.get_or_create(
+                attendee=attendee,
+                name=detail['name'],
+                accessibility_requirements=detail['access_needs'],
+                dietary_restrictions=detail['food_needs'],
+                lca_announce=False,
+                linux_australia=False,
+            )
+
+            print("%s profile %s" % (email, created_or_found(created)))
+
+            ticket_purchased = False
+            for invoice in Invoice.objects.filter(user=user, status=Invoice.STATUS_PAID):
+                for lineitem in invoice.lineitem_set.filter(product=djangoticket):
+                    ticket_purchased = True
```

Comment:
Possibly? My feeling is that we want all the django girls attendees to have a djangogirls ticket. they may well have a student ticket or whatever, in which case they get two badges or whatever.